samples/meeting-av-filters/csharp/MeetingAVFilter/AppPackage/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "version": "1.0.0",
+  "id": "<<GUID ID>>",
+  "packageName": "com.microsoft.teams.avfilter",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://<<BASE-URL>>",
+    "privacyUrl": "https://<<BASE-URL>>/privacy",
+    "termsOfUseUrl": "https://<<BASE-URL>>/termsofuse"
+  },
+  "icons": {
+    "color": "icon-color.png",
+    "outline": "icon-outline.png"
+  },
+  "name": {
+    "short": "Teams Meeting AV Filter",
+    "full": "Teams Meeting AV Filter"
+  },
+  "description": {
+    "short": "This sample demonstrates use of AV filters for teams meetings",
+    "full": "This sample demonstrates use of AV filters for teams meetings by processing the teams meeting video frames."
```

Comment:
This sample demonstrates use of AV filters for Teams meetings by processing the Teams meeting video frames.